pages/common/sort.md

```diff
@@ -13,3 +13,11 @@
 - Sort passwd file by the 3rd field:
 
 `sort -t: -k 3n /etc/passwd`
+
+- Sort a file as number in ascending order:
+
+`sort -n {{filename}}`
+
+- Sort a file and delete duplicate lines:
```

Comment:
Do you mean a suggestion for a different wording, or as in how to perform such a change using git?